tree-sitter-core.el

```diff
@@ -3,14 +3,17 @@
 ;; Copyright (C) 2019  Tuấn-Anh Nguyễn
 ;;
 ;; Author: Tuấn-Anh Nguyễn <ubolonton@gmail.com>
+;;         Jorge Javier Araya Navarro <jorgejavieran@yahoo.com.mx>
 
 ;;; Commentary:
 
 ;; This file contains the core functionalities of tree-sitter.
 ;;
 ;;; Code:
 
-(require 'tree-sitter-dyn)
+(if (functionp 'module-load)
+    (require 'tree-sitter-dyn nil t)
+  (error "dynamic module feature not available, please compile Emacs --with-modules option turned on"))
```

Comment:
Emacs error messages should start with a capitalized word.

Reply:
roger